### PR TITLE
CLI for rust test runner

### PR DESCRIPTION
**Pull request by cptartur**

### CODE REVIEW

File: crates/cairo-lang-protostar/src/test_collector/mod.rs [hunk 1]
```diff
@@ -292,12 +299,12 @@ pub fn collect_tests(
 
     let sierra_program = replace_sierra_ids_in_program(db, &sierra_program);
 
-    let mut builtins = vec![];
-    if let Some(unwrapped_builtins) = maybe_builtins {
-        builtins = unwrapped_builtins.iter().map(|s| s.to_string()).collect();
+    let mut builtins2 = vec![];
```
Comment (karol-bisztyga) [inline crates/cairo-lang-protostar/src/test_collector/mod.rs:302]: pls fix this name, `___2` doesn't indicate anything

File: crates/cairo-lang-protostar/src/test_collector/mod.rs [hunk 2]
```diff
@@ -210,12 +215,19 @@ fn extract_panic_values(db: &dyn SyntaxGroup, attr: &Attribute) -> Option<Vec<Fe
         .collect::<Option<Vec<_>>>()
 }
 
+// TODO docs
```
Comment (piotmag769) [inline crates/cairo-lang-protostar/src/test_collector/mod.rs:218]: TODO

File: crates/cairo-lang-protostar/src/test_collector/mod.rs [hunk 3]
```diff
@@ -227,23 +239,18 @@ pub fn collect_tests(
...
-    {
-        Some((_crate_path, crate_name)) => crate_name,
-        None => "",
-    };
-
-    let main_crate_ids =
-        setup_project_without_cairo_project_toml(db, Path::new(&input_path), main_crate_name)
-            .with_context(|| format!("Failed to setup project for path({})", input_path))?;
+    let main_crate_ids = setup_project(db, Path::new(&input_path))
+        .with_context(|| format!("Failed to setup project for path({})", input_path))?;
```
Comment (piotmag769) [inline crates/cairo-lang-protostar/src/test_collector/mod.rs:243]: Good idea, I forgot we always pass a single file as `input_path` to this function

**Review by piotmag769 — APPROVED** [review]
Really good code cleanup, good job!

**Pull request merged**